vms/administrator/tests/test_report.py

```diff
@@ -2,13 +2,13 @@
 import json
 
 from selenium import webdriver
+from selenium.webdriver.firefox.options import Options
+from selenium.common.exceptions import NoSuchElementException
```

Comment:
This import required? And where? Because the other files don't have this import.

Reply:
This was already imported earlier. I've just rearranged it.